internal/connect/registry_auth_test.go

```diff
@@ -54,7 +54,7 @@ func mockWriteFile(t *testing.T, matcherfile string) {
 
 func mockMkDirAll(t *testing.T) {
 	mkDirAll = func(_ string, perm os.FileMode) error {
-		if perm != 0777 {
+		if perm != 0775 {
```

Comment:
out of curiosity: shouldn't it be `0755`?

Reply:
fixed it and now its `0755`